package.json

```diff
@@ -83,7 +83,6 @@
     "chai-as-promised": "^6.0.0",
     "child-process-debug": "0.0.7",
     "chokidar": "^2.1.6",
-    "chromedriver": "^2.46.0",
```

Comment:
we should add this as peerDependency if it's a required dependency of the parent project